src/components/Login/components/AuthErrorsContainer/AuthErrorsContainer.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import _ from 'lodash';
+
+import styles from './AuthErrorsContainer.css';
+
+export default class AuthErrorsContainer extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.object).isRequired,
+  };
+
+  getErrorMessage(error = {}) {
+    const {
+      code = 'default.error',
+      parameters = [],
+      type = 'error'
+    } = error;
+
+    const values = parameters.reduce((res, { key, value }) => ({ ...res, [key]: value }), {});
+
+    return (
+      <li
+        key={`${code}-${type}`}
+      >
+        <FormattedMessage
+          id={`stripes-core.loginServices.login.${code}`}
+          values={values}
+        />
+      </li>
+    );
+  }
+
+  parseErrors() {
+    const { errors } = this.props;
+    const [messages, className] = _.isEmpty(errors)
+      ? [
+        this.getErrorMessage(),
+        styles.AuthErrorsListNotStyled
+      ] : [
+        errors.map(
+          this.getErrorMessage
+        ),
+        errors.length === 1
+          ? styles.AuthErrorsListNotStyled
+          : styles.AuthErrorsList
+      ];
+
+    return (
+      <ul
+        className={className}
+      >
+        {messages}
+      </ul>
+    );
+  }
+
+  render() {
+    return (
+      <div
```

Comment:
In case when element has only one property please put it in one line `<div className="class">`
Move to new line only when you have more than one property. Fix all the issues in this file.